script/npm_link_foreman_js.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script replace the npm installation of `foreman-js`
+# with your local version. Usefull when developing `foreman-js`
+# Read more about foreman-js: https://github.com/theforeman/foreman-js
+#
+# This script designed to run using `npm run foreman-js:link` in foreman root
+
+set -e
+
+if [[ -z "${FOREMAN_JS_LOCATION}" ]]; then # FOREMAN_JS_LOCATION is empty
+  FOREMAN_JS_LOCATION="../foreman-js"
+  echo "FOREMAN_JS_LOCATION is not defined, using \"${FOREMAN_JS_LOCATION}\" instead"
+elif [ ! -d "${FOREMAN_JS_LOCATION}" ]; then
+  echo "Can't find folder ${FOREMAN_JS_LOCATION}"
+  exit 1
+fi
+
+FOREMAN_JS_LOCATION="../${FOREMAN_JS_LOCATION}"
```

Comment:
this will make it `../../foreman-js` with the default, and add a `../` to the path given by the user which would be confusing

Reply:
imo it will make it easier because you will define `FOREMAN_JS_LOCATION` relatively to foreman root so you can have:
```
home/foreman
home/foreman-js
```

So withing foreman you can run:
```sh
FOREMAN_JS_LOCATION="../foreman-js" npm run foreman-js:lint
```

imo it will be confusing if you will need to set `FOREMAN_JS_LOCATION` relatively to the scripts folder.

Reply:
didn't you reply below that this script runs from the foreman root because it is executed by npm? in that case `../../foreman-js` would be the wrong folder?

Reply:
I think there is a difference in how the `rm` and `ln` commands are working and their parameters.
You can try the script, I have just verified it is working.

Reply:
I think it is only the second parameter of the `ln` that need to be relative to root.

Reply:
ah, i see, it's is because ln creates the link relative to the link location, not to the cwd. you could unite this line with the next but not critical.